### PR TITLE
remove move action if namespace has quota

### PR DESCRIPTION
**Pull request by StrongMonkey**

### CODE REVIEW

**Comment by StrongMonkey** [comment]
https://github.com/rancher/rancher/issues/15547

File: pkg/api/customization/namespace/namespace.go [hunk 1]
```diff
@@ -93,7 +93,8 @@ func NewFormatter(next types.Formatter) types.Formatter {
 			next(request, resource)
 		}
 		annotations := convert.ToMapInterface(resource.Values["annotations"])
-		if convert.ToString(annotations[helm.AppIDsLabel]) == "" {
+		_, hasQuota := resource.Values["resourceQuota"]
+		if convert.ToString(annotations[helm.AppIDsLabel]) == "" && !hasQuota {
```
Comment (alena1108) [inline pkg/api/customization/namespace/namespace.go]: we should check the resourceQuota presence field on the project you are moving to, not on the namespace

**Comment by alena1108** [comment]
LGTM

**Comment by cjellick** [comment]
LGTM

**Pull request merged**